src/webui/btjson.cpp

```diff
@@ -370,6 +373,7 @@ namespace
         if (torrent->hasMetadata())
             ret[KEY_TORRENT_FIRST_LAST_PIECE_PRIO] = torrent->hasFirstLastPiecePriority();
         ret[KEY_TORRENT_CATEGORY] = torrent->category();
+        ret[KEY_TORRENT_TAGS] = getTagsForTorrent(torrent);
```

Comment:
nagging again... since the function is reduced to one line, why not just inline it here?

Reply:
Totally agree. Keep up the nagging!  😉